emily_cron/app/models/deposit.py

```diff
@@ -48,16 +48,21 @@ def from_json(cls, j: dict[str, Any]) -> Self:
     @property
     def lock_time(self) -> int:
         """Extracts lock time from reclaim script."""
-        lock_time = script.parse(self.reclaim_script)[0]
-        if lock_time.startswith("OP_"):
-            return int(lock_time[len("OP_") :])
-        return int.from_bytes(bytes.fromhex(lock_time), byteorder="little")
+        script = Script.parse(self.reclaim_script)
+        op_code_maybe = script.view().split()[0]
```

Comment:
nit: using `as_list=True` in `view` we can avoid join+split (and be sure there's nothing messing up the splitting); also below